### PR TITLE
NT tweaks

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

In "NeoTheology Bio-Fabric Designs",

Added "reinforced shoes" design, costs 25 biomatter,
Added "custodian's jackboots" design, costs 35 biomatter,
Added "cruciformed backpack" design, costs 30 biomatter and 2 plastic,
Added "cruciformed satchel" design, costs 16 biomatter and 2 plastic,

Removed "leather satchel" design,
Removed "champion belt" design,
Removed "utility belt" design,
Removed "tactical belt" design,
Removed "cash bag" design.

Fixed typo in valkyrie disc's path.
Neotheologian tactical belt now can hold Svalinn laser pistol.
Added shoes to NT lockers. Custodian's jackboots for custodians and reinforced shoes for everyone else.
Added "spare ID" box to preacher's locker.
Deleted unused duplicate of preacher's locker.
Replaced one aster-brand S-cell with two NT-branded S-cells in NT lockers. Plasma version for preacher.

image

## Why It's Good For The Game

Played as Last Shelter's disciple today. It's pretty sad to not have shoes. And ID.
Would have sense for NT to make NT-branded apparel, instead of generic. Same for cells.
Also odd that NT-made belt can hold FS-made pistol, but can't hold NT-made pistol.
Cash bag and champion belt is at least 9 years old crap that comes from Bay 12. It have no reason to be on NT disc.

## Changelog
:cl:
add: shoes and bags designs to NT bio-fabric disc
add: spare IDs box in preacher locker
del: bay leftover designs from NT bio-fabric disc
tweak: replaced generic cells in NT lockers with NT-manufactured cells
tweak: NT tactical belt now can hold svalinn.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 1]
```diff
@@ -251,7 +246,7 @@
 		/datum/design/autolathe/cell/medium/high,
 	)
 
-/obj/item/computer_hardware/hard_drive/portable/design/guns/nt_valkirye
+/obj/item/computer_hardware/hard_drive/portable/design/guns/nt_valkyrie
```
Comment (TheShown911) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:249]: If this path is used anywhere else it will need to be updated.

**Review by TheShown911 — APPROVED** [review]
Please expand the changelog for the devlog.

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm (hunk 1 above)
Comment (SirRichardFrancis) [reply code/game/objects/items/weapons/design_disks/NeoTheology.dm:249]: It's not.

**Review by drexample — DISMISSED** [review]
Approval, but the changelog is in the wrong format, please fix

**Pull request merged**